pkg/didcomm/protocol/transport/http/http_transport.go

```diff
@@ -0,0 +1,262 @@
+/*
```

Comment:
transport is a sibling, not a child of protocol.
https://wiki.hyperledger.org/display/ARIES/Framework+Go+Package+Hierarchy

Reply:
moved transport at the same level as protocol